org-roam-buffer.el

```diff
@@ -108,8 +108,9 @@ When non-nil, the window will not be closed when deleting other windows."
             (key-backlinks (org-roam--get-backlinks (s-chop-prefix "cite:" roam-key)))
             (grouped-backlinks (--group-by (nth 0 it) key-backlinks)))
       (progn
-        (insert (format "\n\n* %d Cite backlinks\n"
-                        (length key-backlinks)))
+        (insert (let ((l (length key-backlinks)))
+                  (format "\n\n* %d Cite backlink%s\n"
+                          l (if (> l 1) "s" ""))))
```

Comment:
Better implemented as a helper function (org-roam-pluralize) or something.

Reply:
Sure, on it.

Reply:
Uhm, I'm actually not sure how to best implement this.  db6a477360e9bef930a92e48f434f9f7bd557a5f is hardly any better.  I'm thinking about a double `format` expansion with `%%s` for the plurals.